modules/node_modules/@ncigdc/utils/generated-relay-query-parts/repositoryCaseAggregations.js

```diff
@@ -0,0 +1,1868 @@
+  shouldShow_samples__portions__analytes__spectrophotometer_method: false,
+  shouldShow_samples__portions__analytes__submitter_id: false,
+  shouldShow_samples__portions__analytes__updated_datetime: false,
+  shouldShow_samples__portions__analytes__well_number: false,
+  shouldShow_samples__portions__center__center_id: false,
+  shouldShow_samples__portions__center__center_type: false,
+  shouldShow_samples__portions__center__code: false,
+  shouldShow_samples__portions__center__name: false,
+  shouldShow_samples__portions__center__namespace: false,
+  shouldShow_samples__portions__center__short_name: false,
+  shouldShow_samples__portions__created_datetime: false,
+  shouldShow_samples__portions__creation_datetime: false,
+  shouldShow_samples__portions__is_ffpe: false,
+  shouldShow_samples__portions__portion_id: false,
+  shouldShow_samples__portions__portion_number: false,
+  shouldShow_samples__portions__slides__created_datetime: false,
+  shouldShow_samples__portions__slides__number_proliferating_cells: false,
+  shouldShow_samples__portions__slides__percent_eosinophil_infiltration: false,
+  shouldShow_samples__portions__slides__percent_granulocyte_infiltration: false,
+  shouldShow_samples__portions__slides__percent_inflam_infiltration: false,
+  shouldShow_samples__portions__slides__percent_lymphocyte_infiltration: false,
+  shouldShow_samples__portions__slides__percent_monocyte_infiltration: false,
+  shouldShow_samples__portions__slides__percent_necrosis: false,
+  shouldShow_samples__portions__slides__percent_neutrophil_infiltration: false,
+  shouldShow_samples__portions__slides__percent_normal_cells: false,
+  shouldShow_samples__portions__slides__percent_stromal_cells: false,
+  shouldShow_samples__portions__slides__percent_tumor_cells: false,
+  shouldShow_samples__portions__slides__percent_tumor_nuclei: false,
+  shouldShow_samples__portions__slides__section_location: false,
+  shouldShow_samples__portions__slides__slide_id: false,
+  shouldShow_samples__portions__slides__submitter_id: false,
+  shouldShow_samples__portions__slides__updated_datetime: false,
+  shouldShow_samples__portions__submitter_id: false,
+  shouldShow_samples__portions__updated_datetime: false,
+  shouldShow_samples__portions__weight: false,
+  shouldShow_samples__preservation_method: false,
+  shouldShow_samples__sample_id: false,
+  shouldShow_samples__sample_type: false,
+  shouldShow_samples__sample_type_id: false,
+  shouldShow_samples__shortest_dimension: false,
+  shouldShow_samples__submitter_id: false,
+  shouldShow_samples__time_between_clamping_and_freezing: false,
+  shouldShow_samples__time_between_excision_and_freezing: false,
+  shouldShow_samples__tissue_type: false,
+  shouldShow_samples__tumor_code: false,
+  shouldShow_samples__tumor_code_id: false,
+  shouldShow_samples__tumor_descriptor: false,
+  shouldShow_samples__updated_datetime: false,
+  shouldShow_submitter_id: false,
+  shouldShow_updated_datetime: false,
+};
```

Comment:
Can now run `/modules/node_modules/@ncigdc/utils/generated-relay-query-parts/generate-script.js` to genreate these files :D